solutions/standard/main.tf

```diff
@@ -0,0 +1,39 @@
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.4"
+  resource_group_name          = var.existing_resource_group == false ? var.resource_group_name : null
+  existing_resource_group_name = var.existing_resource_group == true ? var.resource_group_name : null
+}
+
+########################################################################################################################
+# Event Notifications
+########################################################################################################################
+
+locals {
+  service_credential_names = var.service_credential_names
+  # KMS Add on
+  existing_kms_instance_crn = var.existing_kms_instance_crn != null ? var.existing_kms_instance_crn : null
+  kms_endpoint_url          = var.kms_encryption_enabled ? var.kms_endpoint_url != null ? var.kms_endpoint_url : format("https://%s.kms.cloud.ibm.com", var.region) : null
```

Comment:
hmm, this means if user does not pass a value for `kms_endpoint_url`, it will default to a public endpoint. Not sure we should do that, as eventually KMS DA will default to private only. Suggest to just make `kms_endpoint_url` a required variable. The input can be got from the KMS DA output anyway